supervisor/addons/model.py

```diff
@@ -102,6 +102,16 @@ def is_installed(self) -> bool:
     def is_detached(self) -> bool:
         """Return True if add-on is detached."""
 
+    @property
+    def available_arch(self) -> bool:
+        """Return True if the running arch is supported by the add-on."""
+        return self._available_arch(self.data)
+
+    @property
+    def available_homeassistant(self) -> bool:
+        """Return True if the running version of Home Assistant is supported by the add-on."""
+        return self._available_homeassistant(self.data)
+
```

Comment:
Let's put this into AddStore model

Reply:
Had to revert it, it's used in restore, so too much would be needed to be moved.